codex-rs/tui/src/app.rs

```diff
@@ -223,9 +223,7 @@ impl App<'_> {
                         } => {
                             match &mut self.app_state {
                                 AppState::Chat { widget } => {
-                                    if widget.on_ctrl_c() {
-                                        self.app_event_tx.send(AppEvent::ExitRequest);
-                                    }
+                                    widget.on_ctrl_c();
```

Comment:
Wait, why are we changing Ctrl-C behavior. Did you manually test this? What is different now?